### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by TheSuperGamer20578**
Grammer was bad in readme. This PR will fix that.

# Checklist

- [ ] This PR has been tested
	- [ ] Everything still works
	- [ ] No major bugs have been made
		- [ ] I have listed minor bugs that have been added
- [x] I have listed all changes that this PR makes
- [x] This PR needs no further work
- [ ] Manual restart and/or configuration required post-merge
- [x] README.md or other documentation will be updated by this PR
- [ ] I have listed how to make bot work again post-merge 

- [ ] This is a major change
- [x] This is a minor change
<p/>

- [ ] This is a bug fix
- [ ] This is an enhancement or new feature
- [x] Other: documentation fix

### CODE REVIEW

**Review by ogxn — DISMISSED** [review]
just one problem: "existant" should be "existent." once that's fixed I can approve

**Comment by TheSuperGamer20578** [comment]
> just one problem: "existant" should be "existent." once that's fixed I can approve

Just one proplem in PR reveiws you should comment kn the lines that have problems so that tye contributer doesnt need to search through the entirety of all the files in a pr

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-4. install requirements from `requirements.txt` this can be done by passing the `-r` flag to pip followed by the requirements file if for some reason you want to use `start.py` you will also have to install `termcolor`
-5. `cd bot` and run `core.py` or you can run `start.py` if you want a bit of a ui but its a bit useless if you are on linux and want to use the auto updating cog you will have to run it with `start.sh`
+1. Clone this repo.
+2. In `bot/` rename `Config.sample` to `Config`.
+3. Again in `bot/` edit `Config/config.ini` and `Config/firebase.json`.
+4. Install requirements from `requirements.txt`. This can be done by passing the `-r` flag to pip followed by the requirements file.
+5. `cd bot` and run `core.py`. If you are on linux and want to use the auto updating cog you will have to run it with `start.sh`.
 
-For more info look at the [Wiki](https://github.com/TheSuperGamer20578/Sudan-bot/wiki)
+For more info look at the (currently non-existant) [Wiki](https://github.com/TheSuperGamer20578/Sudan-bot/wiki)
```
Comment (TheSuperGamer20578) [inline README.md:9]: @ogxn is this what you mean in your review?

Comment (ogxn) [reply README.md:9]: yes but please note this is literally ONE file

Comment (TheSuperGamer20578) [reply README.md:9]: yes but there are NINE lines it could be any one

**Review by ogxn — APPROVED** [review]

**Pull request merged**